src/explore-education-statistics-frontend/src/components/__tests__/Feedback.test.tsx

```diff
@@ -0,0 +1,133 @@
+import Feedback from '@frontend/components/Feedback';
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import _feedbackService from '@frontend/services/feedbackService';
+import { FeedbackRequest } from '@common/services/types/feedback';
+
+jest.mock('@frontend/services/feedbackService');
+
+const feedbackService = jest.mocked(_feedbackService);
+
+describe('Feedback', () => {
+  test('renders initial state correctly', () => {
+    render(<Feedback />);
```

Comment:
Could add a line break after the `render` method to make it more obvious where rendering ends and assertions begin. Currently, there isn't a lot of breathing room and makes it less readable.

Would typically suggest leaving line breaks between any significant 'stages' of a test (i.e. arrange, act and assert stages) to improve readability.